client/js/components/pages/HomePage.js

```diff
@@ -1,10 +1,350 @@
 import React, { Component } from 'react';
+import { Link } from 'react-router';
 
 class HomePage extends Component {
+  constructor () {
+    super();
+    this.state = { visibility: false };
+  }
+
+  onClick () {
+    this.setState({ visibility: true });
+  }
+
   render() {
+    let visibility = this.state.visibility ? 'hidden' : '';
+    // to do: remove hidden class if click in the home logo
     return (
-      <div>
-        <h1>Hello!!</h1>
+      <div className={ 'panel homepage ' + visibility }>
+        <article>
+          <section>
+            <h2>Welcome to X-Map</h2>
+            <p>Please <Link to="/login">Log in</Link> or <Link to="/register">Register</Link> to proceed.</p>
+            <p>An <a href="https://github.com/x-team/x-map/" target="_blank">open source</a> project at <a href="http://x-team.com/community/" target="_blank">X-Team</a></p>
+            <button onClick={ this.onClick.bind(this) }>Click to hide panel.</button>
```

Comment:
All other buttons I saw had className="button", shouldn't we have it here as well?